README.md

```diff
@@ -4,32 +4,32 @@ Drupal source code for the NIDirect website: https://www.nidirect.gov.uk.
 
 ## Getting started
 
-This repository should be cloned as follows inside the /web directory of a working Drupal 8 NIDirect site:
+This repository should be cloned as follows inside the /web/profiles directory of a working Drupal 8 NIDirect site:
 
- git clone git@svegit01.thestables.net:dss/nidirect-d8-test-install-profile.git profiles
+ git clone git@svegit01.thestables.net:dss/nidirect-d8-test-install-profile.git custom
  
-(after doing this, you should have a 'web/profiles/test_profile' directory in your project)
+(after doing this, you should have a 'web/profiles/custom/test_profile' directory in your project)
 
 ## Updating the install profile
 
 Simply export your current config to the install profile directory like this:
 
- lando drush config-export --destination=profiles/test_profile/config/sync
+ lando drush config-export --destination=profiles/custom/test_profile/config/sync
  
-After doing this, edit profiles/test_profile/config/sync/core.extension.yml and make sure that the install profile is set to 'test_profile' rather than 'standard'. Note that this may involve making two changes towards the bottom of the file, as shown in the following sampel from the end of the core.extension.yml file:
+After doing this, edit profiles/custom/test_profile/config/sync/core.extension.yml and make sure that the install profile is set to 'test_profile' rather than 'standard'. Note that this may involve making two changes towards the bottom of the file, as shown in the following sample from the end of the core.extension.yml file:
 
   test_profile: 1000
   theme:
     stable: 0
     classy: 0
     bartik: 0
     seven: 0
-    stark: 0
+    stark: 0 
```

Comment:
See if your IDE can strip trailing whitespace on file save. That helps avoid this kind of thing which can help reduce noise on a diff